src/rust/engine/src/context.rs

```diff
@@ -222,6 +222,47 @@ impl Core {
   pub fn http_client(&self) -> reqwest::r#async::Client {
     self.store_and_command_runner_and_http_client.get().2
   }
+
+  ///
+  /// Start running a Future on a tokio Runtime.
```

Comment:
Somewhere in here we should mention explicitly the reason for the overrides (setting logging state).

Reply:
Done